### PR TITLE
update rand in tink-core

### PR DESCRIPTION
**Pull request by harryfei**

### CODE REVIEW

**Review by daviddrysdale — COMMENTED** [review]
Thanks for this (I'd been hoping that `ed25519-dalek` would get an update and we could move everything to `rand@0.8`, but I guess not&hellip;)

File: Cargo.lock [hunk 1]
```diff
@@ -584,9 +584,9 @@ dependencies = [
 
 [[package]]
 name = "ed25519"
-version = "1.0.2"
+version = "1.5.3"
```
[the remark this answers is not in the document]
Comment (harryfei) [reply Cargo.lock:587]: I will clean them. That's the dirty result when I try to update ed25519-dalek.

**Comment by harryfei** [comment]
ed25519 has only a beta version which depends on rand@0.8 https://github.com/dalek-cryptography/ed25519-dalek#beta. If you accept use this version, I will upgrade it too.

**Comment by harryfei** [comment]
update already.

**Comment by daviddrysdale** [comment]
This still mixes a bunch of different things into a single commit; I'd prefer a bit more separation, so I'm putting #436 together and then the `ed25519-dalek` upgrade can happen separately.

**Comment by harryfei** [comment]
Ok. Just wait that PR.

**Comment by daviddrysdale** [comment]
Could you rebase/regenerate this so it just has the `ed25519-dalek` update (and associated `rand` 0.8 dep) please?

Also, do you know if an official 2.0.0 release is imminent?

**Comment by harryfei** [comment]
Sure. I don't know the plan of ed25519-dalek's 2.0.0 releasing.
Should I just to wait ed25519-dalek's 2.0.0 before rebasing this PR?

**Comment by daviddrysdale** [comment]
Superseded elsewhere, thanks.

**Pull request closed without merging**